.gitmodules

```diff
@@ -66,3 +66,9 @@
 [submodule "platform/broadcom/sonic-platform-modules-accton"]
 	path = platform/broadcom/sonic-platform-modules-accton
 	url = https://github.com/edge-core/sonic-platform-modules-accton.git
+[submodule "src/tacacs/sonic-pam-tacplus"]
+	path = src/tacacs/sonic-pam-tacplus
+	url = https://github.com/liuqu/sonic-pam-tacplus.git
```

Comment:
I can see only one patch is made. It is better maintian it as a patch, like what we did for libteam.

Can you follow that as an example?

https://github.com/Azure/sonic-buildimage/tree/master/src/libteam 


ttps://github.com/liuqu/sonic-pam-tacplus/commit/6aab5b6cafb65763e297f99e889418677528cd35